vertica-kafka-scheduler/README.md

```diff
@@ -1,58 +1,303 @@
     -v $PWD/vkafka-log-config-debug.xml:/opt/vertica/packages/kafka/config/vkafka-log-config.xml \
     -v $PWD/log:/opt/vertica/log \
     --user $(perl -E '@s=stat "'"$PWD/log"'"; say "$s[4]:$s[5]"') \
-        vkconfig launch --conf /etc/vkconfig.conf
-```
-
-## Example
-`example.sh` is a full example of configuring and running a scheduler.  It uses
-docker-compose to create a kafka and vertica service, then it posts some kafka
-JSON messages, waits, then shows the messages in vertica tables.
-
-## Building docker container
-
-This is how you rebuild the docker image to customize it or update it and test it.
-This requres the vertica package installed in /opt/vertica or specified in the `make`
-command with `VERTICA_INSTALL=/wherever/opt/vertica`
-1. Clone this repository.
-2. Open a terminal in the `vertica-kafka-scheduler` directory.
-3. Install Vertica (or use rpm2cpio vertica.rpm |cpio -idmv and export VERTICA_INSTALL=./opt/vertica)
-4. Build the kafka scheduler container with
-    ```
-    make build
-    ```
-
-## Push to docker hub
-
-1. Clone this repository.
-2. Open a terminal in the `vertica-kafka-scheduler` directory.
-3. Install Vertica (or use rpm2cpio vertica.rpm |cpio -idmv and export VERTICA_INSTALL=./opt/vertica)
-4. Build and push containers
-    ```
-    VERTICA_VERSION=latest make push
-    ```
+        vkconfig launch --conf /etc/vkconfig.conf &
+```
+Additionally, the preceding command does the following:
+- Defines the `--user` with a Perl script that extracts the `/log` file owner and group information, and then formats those values in `user:group` format.
+- Uses the `&` operator to execute `vkconfig launch` as a background process.
+
+### Building a custom scheduler container
+
+In some circumstances, you might want to build a custom vertica/kafka-scheduler container. This repository provides a [Makefile](./Makefile) with targets that accept build variables to simplify the build process.
+
+#### Prerequisites
+- Vertica binary or rpm2cpio vertica.rpm | cpio -idmv and export VERTICA_INSTALL=./opt/vertica
+- Java libraries located in `/vertica/java`.
+
+For additional information about Vertica and Java development, see [Java SDK](https://www.vertica.com/docs/latest/HTML/Content/Authoring/ExtendingVertica/Java/DevelopingInJava.htm) in the Vertica documentation.
+
+#### `build` make target
+
+Use the `build` target to create a custom container. Depending on your Vertica environment, you might need to include build variables described in the following table:
+
+| Variable        | Description |
+|:----------------|:------------|
+| VERTICA_INSTALL | The location of your Vertica binary installation. Define this variable if you want to copy the local install of the Java libraries.<br>**Default**: `/opt/vertica` |
+| VERTICA_VERSION | The Vertica version that you want to use to build the scheduler container. The scheduler version must match the Vertica database version. <br>**Default**: `latest` |
```

Comment:
default = whatever version VERTICA_INSTALL is pointing to.

Reply:
Updated.